apiStar_folder/templates/index1.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <body>
+
+        <h2>Textarea for user's input</h2>
+        <p>Type input in env.spec format.</p>
+        <textarea id="myTextarea"></textarea>
+
+        <button type="button" onclick="myFunction()">Try it</button>
```

Comment:
Instead of using the `onclick` HTML attribute, we should prefer doing it with plain JavaScript:

1. Access the button with JavaScript
2. Use `addEventListener` to register the intended behaviour